tasks/redhat.yml

```diff
@@ -17,3 +17,12 @@
     state: present
   notify:
     - restart filebeat
+
+- name: modify filebeat service file to include environment variables
```

Comment:
This would probably make sense as a drop-in unit instead of modifying the unit directly: https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Description

> Along with a unit file foo.service, a "drop-in" directory foo.service.d/ may exist. All files with the suffix ".conf" from this directory will be parsed after the unit file itself is parsed. This is useful to alter or add configuration settings for a unit, without having to modify unit files.